### PR TITLE
PXBLUE-2653 Rebranded highcharts demos

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # PXBLUE-2653 

#### Changes proposed in this Pull Request:
- pxblue ==> brightlayer-ui

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,4 +42,4 @@ Breaking changes:
 
 ### v1.0.0 (October 26, 2018)
 
--   This library has been ported from @pxblue/visualizations
+-   This library has been ported from @brightlayer-ui/brightlayer-ui
```
Comment (daileytj) [inline CHANGELOG.md]: The changelog history shouldn't change here. The library for that changelog entry was still ported from @pxblue/visualizations, please revert this change.

File: react-demo/src/Logo.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 
-type PXBlueSVGProps = {
+type BrightlayerUISVGProps = {
     className?: string;
     color?: string;
     size?: number;
 };
-export const PXBlueSVG: React.FC<PXBlueSVGProps> = (props) => {
+export const BrightlayerUISVG: React.FC<BrightlayerUISVGProps> = (props) => {
```
Comment (daileytj) [inline react-demo/src/Logo.tsx]: This is still a PXBlue logo so we don't need to change the name here. It doesn't look like it's being used anywhere in the demo so I'm tempted to just rip it out entirely. Thoughts? @emclaug2 @joebochill

File: CHANGELOG.md (hunk 1 above)
Comment (bkarambe) [reply CHANGELOG.md]: Done.

File: react-demo/src/Logo.tsx (hunk 2 above)
Comment (daileytj) [reply react-demo/src/Logo.tsx]: For the sake of time, just revert the name change. We can rip it out later if we want.

**Comment by daileytj** [comment]
You're still using the highcharts beta package in the demos. Please update this to use the latest release.

File: angular-demo/src/app/components/pxb-logo/pxb-logo.component.ts [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import { Component, Input } from '@angular/core';
 
 @Component({
-    selector: 'app-pxb-logo',
```
Comment (daileytj) [inline angular-demo/src/app/components/pxb-logo/pxb-logo.component.ts:4]: This component icon can just be removed as well.

**Review by daileytj — APPROVED** [review]
